diff --git a/mc-rs/src/world/items.rs b/mc-rs/src/world/items.rs
index 69e7942..da40b92 100644
--- a/mc-rs/src/world/items.rs
+++ b/mc-rs/src/world/items.rs
@@ -30,6 +30,8 @@ pub const STEW: u16 = 282;
 pub fn heal_amount(id: u16) -> Option<i32> {
     match id {
         STEW => Some(10),
+        319 => Some(3), // raw porkchop
+        320 => Some(8), // cooked porkchop
         _ => None,
     }
 }
diff --git a/mc-rs/src/world/mod.rs b/mc-rs/src/world/mod.rs
index 87bac8e..8daf5d2 100644
--- a/mc-rs/src/world/mod.rs
+++ b/mc-rs/src/world/mod.rs
@@ -7,6 +7,7 @@ pub mod pick;
 pub mod sky;
 pub mod dig;
 pub mod items;
+pub mod mobs;
 pub mod names;
 pub mod craft;
 pub mod vitals;
